### ISSUE TITLE
GET parameter condition is case sensitive

### ISSUE THREAD

**Issue by contolini** [cfpb/django-flags#30]
Currently, a [GET parameter](https://cfpb.github.io/django-flags/conditions/#parameter) will only trigger a flag if it equals `True`. For example, given:

```
FLAGS = {
    'THERE_IS_AN_OUTAGE': [
        {'condition': 'parameter', 'value': 'outage'}
    ]
}
```

The flag will be enabled with `/foo?outage=True`.

It won't be enabled with `/foo?outage=true`, `/foo?outage=today` or `/foo?outage`.

I'm unsure if the above functionality is by design but IMHO it should at least accept `/foo?outage=true` if not the other options as well.

BTW A+ Django app 10/10 five stars would download again 💯

**Comment by willbarton** [comment]
I'm unsure if the above functionality is by design but IMHO it should at least accept /foo?outage=true if not the other options as well.

@contolini That's a good question — the git history and the documentation are somewhat ambiguous, and I suspect the person responsible *cough* doesn't recall either.

What would be the desired behavior here?

> It won't be enabled with /foo?outage=true, /foo?outage=today or /foo?outage.

I kinda think it should be enabled for any of these, personally — that the parameter exists should be enough. I'll open a PR to fix this and #27 together, but I'd like thoughts on this question specifically.

**Comment by contolini** [comment]
> I kinda think it should be enabled for any of these

👍 I agree, I think the existence of the parameter should enable the flag.
